### DISCUSSION TITLE
Future toCSV?

### DISCUSSION THREAD

**Discussion by ehrev** [JonathanVusich/pcpartpicker#50] [Ideas]
This isn't an issue, just inquiring about a future feature addition. I am looking to generate a CSV dataset of computer hardware components as everything I can find online is quite limited. This API seems promising but I don't have much use for the JSON. I saw you mentioned in a past issue thread that a toCSV seems more niche but is it a feature you would add?

**Comment by JonathanVusich** [comment]
Sure, this shouldn't be too hard to implement. I can look into it.

**Comment by JonathanVusich** [comment]
It is not immediately clear to me how the data would be organized in the CSV format. Do you have a specific idea of how you think it should be formatted?

**Comment by ehrev** [comment]
Let me take a look at the current data being output into the JSON to get a more detailed idea of how I would envision a CSV format. Thanks for getting back to me!
